mcu_soc/design/sim/doit_build.py

```diff
@@ -4,12 +4,13 @@
 
 
 OUTPUT_DIR  = "./build/sim"
-SOURCE_DIR  = importlib.resources.files("my_design") / "sim"
+COMMON_DIR  = importlib.resources.files("common") / "sim"
+SOURCE_DIR  = importlib.resources.files("mcu_soc") / "design/sim"
```

Comment:
Doesn't need fixing for this PR, but this will fail on non-FS importlib backends